gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/work/assistance/Help.java

```diff
@@ -77,9 +77,9 @@ public static String qualifyNamePerExecWithoutFlowExecId(String name, Config wor
   }
 
   /** @return execution-specific name, incorporating any {@link ConfigurationKeys#FLOW_EXECUTION_ID_KEY} from `workerConfig` */
-  public static String qualifyNamePerExecWithFlowExecId(String name, Config workerConfig) {
-    Optional<String> optFlowExecId = Optional.ofNullable(ConfigUtils.getString(workerConfig, ConfigurationKeys.FLOW_EXECUTION_ID_KEY, null));
-    return name + "_" + calcPerExecQualifierWithOptFlowExecId(optFlowExecId, workerConfig);
+  public static String qualifyNamePerExecWithFlowExecId(String name, Config jobProps) {
```

Comment:
javadoc still names `workerConfig`.  also, we could be more agnostic and just call it `config`, so it's up to the caller of what to pass in, since really you haven't altered impl semantics here, only naming.